source/_templates/registrations/common/set_agent_name.rst

```diff
@@ -0,0 +1,7 @@
+.. Copyright (C) 2020 Wazuh, Inc.
+
+.. note::
+
+ If the new agent’s name is not provided, it is set automatically using hostname. To specify the agent's name add ``-A <agent_name>`` to the command below.
```

Comment:
This comment must not be a Note. Please include it after the command as normal text.

Reply:
Done